src/utils/calculateEstimatedPrizePool.ts

```diff
@@ -0,0 +1,46 @@
+import { ethers, BigNumber, constants } from 'ethers';
+
+import {
+  normalizeTo18Decimals,
+  DEFAULT_TOKEN_DECIMAL_PRECISION as DEFAULT_TOKEN_PRECISION,
+} from './normalizeTo18Decimals';
+
+const bn = BigNumber.from;
+
+const SECONDS_PER_BLOCK = 14;
+
+type CalculateEstimatedPoolPrizeArgs = {
+  tokenDecimals: number;
+  awardBalance: BigNumber;
+  poolTotalSupply: BigNumber;
+  supplyRatePerBlock: BigNumber;
+  prizePeriodRemainingSeconds: BigNumber;
+};
+
+export function calculateEstimatedPoolPrize({
+  tokenDecimals,
```

Comment:
got this from here: https://github.com/pooltogether/pooltogether-reference-pool-ui/blob/master/lib/utils/calculateEstimatedPoolPrize.js